platform/ui/src/components/Table/Table.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+const Table = ({ children, className, fullWidth, style }) => {
+  const classes = {
+    base: 'text-lg text-white',
+    fullWidth: {
+      true: 'w-full',
+      false: '',
+    },
+  };
+
+  return (
+    <div
+      className={classnames(
+        classes.base,
+        classes.fullWidth[fullWidth],
+        className
+      )}
+      style={style}
+    >
+      {children}
+    </div>
+  );
+};
+
+Table.defaultProps = {
```

Comment:
I think this is the standard/recommended approach.

Reply:
Hmm I have asked because I've seeing 3 ways in the code for ohif/ui right now. 
I will create a new PR to make them all using `Component.defaultprops = {}`.